### DISCUSSION TITLE
Where do posts show up once published?

### DISCUSSION THREAD

**Discussion by mayanayza** [GetPublii/Publii#1682] [Help]
Does Publii automatically list posts somewhere or is this something that needs to be configured manually?

If the former, where? I have a homepage set up but nothing is displaying there.

If the latter, how? Docs aren't super clear to me on this.

I have three sample posts that I've created but they are nowhere to be seen. Using the default Simple theme.

**Comment by bobmitro** [comment]
Have you published the posts or are they in draft status?

**Reply by mayanayza** [reply]
They are all published

**Comment by GetPublii** [comment]
One more questions: do you have three posts or pages? Because only posts
are displayed as a loop on the homepage.

**Reply by mayanayza** [reply]
I have three posts; they aren't showing up anywhere

**Reply by bobmitro** [reply]
It's very strange. If it is a testing site, attach your site backup here or send it to me at the bob...

**Reply by mayanayza** [reply]
I've just sent it to you, thanks!

**Comment by bobmitro** [comment]
Ok, I got it. You don't see posts on the homepage because you set the "Page as the homepage" in the Site Settings, and your selected here "Portfolio" page is empty (there is no content, only a title). 
So, just uncheck this checkbox to see the posts on the homepage.

<img width="940" alt="screenshot 2024-09-11 at 19 09 18" src="https://github.com/user-attachments/assets/8a4c8dbd-e3c2-4adc-80e7-234ba13762fd">

**Reply by mayanayza** [reply]
Thanks!

If I wanted to have my posts show up on a specific page rather than the homepage (that was my intent in setting this), how would I do so?

**Reply by bobmitro** [reply]
Just set the Post prefix (Site Settgins - URLs). Then, the "Portfolio" page will be displayed as the homepage, and the post listing will be available under the URL specified by the Post prefix.

**Reply by mayanayza** [reply]
If I'd rather have a homepage without posts listed at all, in addition to a separate portfolio page (which it sounds like I can do, thanks!) is this possible?

**Reply by bjazmoore** [reply]
Here is how that works: https://github.com/GetPublii/Publii/discussions/1653#discussioncomment-10522865

**Reply by jdtoppin** [reply]
The link to add a menu item works; however, in themes with a filter, the filter doesn't work on the separate index page. I'm guessing you have to setup a page override or something in my limited reading on it, but it would be nice if this was baked into themes.

**Reply by bobmitro** [reply]
> the filter doesn't work on the separate index page

Which theme are you using?

**Reply by jdtoppin** [reply]
> > the filter doesn't work on the separate index page
> 
> Which theme are you using?

This is on Mercury. Works fine when the home page is the posts page, but when you use the menu to have a separate posts-index, seems like the js or something isn't loaded.

**Reply by bobmitro** [reply]
Yep, there was a bug! Grab the new version  [3.1.1.0](https://marketplace.getpublii.com/themes/mercury/) – it fixes this issue right up.
